doc/prepare_raspberrypi.rst

```diff
@@ -1,5 +1,22 @@
-Install Raspberry Pi OS on the Raspberry Pi (by using a Linux laptop)
-=====================================================================
+Installation and configuration of Raspberry Pi
+==============================================
+The p-net stack and sample application
+has been tested with:
+* Raspberry Pi 3 Model B+
+
+To avoid problems it is recomended to start with a fresh
```

Comment:
recommended

Reply:
Done, also enabled spelling check for .rst files..